### PR TITLE
[CodeStyle] Format gherkin code to be more consistent

### PR DESCRIPTION
**Pull request by malukenho**
I used the [malukenho/kawaii-gherkin](https://github.com/malukenho/kawaii-gherkin) to format the code.

### CODE REVIEW

File: features/backend/groups.feature [hunk 1]
```diff
@@ -45,7 +45,7 @@ Feature: Customer groups management
     Scenario: Creating group
         Given I am on the group creation page
          When I fill in the following:
-            | Name  | Dealers                   |
+            | Name | Dealers |
```
Comment (michalmarcinkowski) [inline features/backend/groups.feature]: :+1: for tables corrections.

File: features/backend/metadata/metadata_customization.feature [hunk 2]
```diff
@@ -1,75 +1,75 @@
...
-      | Title       | Lorem ipsum       |
-      | Keywords    | sylius, ecommerce |
-      | Description | <empty>           |
-      | Twitter     | <empty>           |
+    Scenario: Creating page metadata without Twitter metadata
+        Given I am customizing metadata with identifier "FOO-BAR"
+         When I fill in "Title" with "Lorem ipsum"
+          And I fill in "Keywords" with "sylius, ecommerce"
+          And I press "Save changes"
+         Then I should be on the page of metadata container with id "FOO-BAR"
```
Comment (michalmarcinkowski) [inline features/backend/metadata/metadata_customization.feature]: We shouldn't align the steps to the right, they should be like natural sentences aligned to left like before that change.

**Comment by michalmarcinkowski** [comment]
@malukenho is there a way we could run it only for tables? Or can we adjust the steps alignment to the left?

**Comment by malukenho** [comment]
@michalmarcinkowski yes, I can run it only for tables, but actually indentation is mixed on feature files, if the "default" that you want is align to left, I can do it.

> Note: I'm finishing an format validator that can be put on travis build if wanted.

**Comment by pjedrzejewski** [comment]
@malukenho Wow, this is really cool! Yes, please align everything to left. Validator sounds interesting too. Nice work. :)

**Comment by malukenho** [comment]
@michalmarcinkowski @pjedrzejewski done :smile:

**Comment by michalmarcinkowski** [comment]
@malukenho awesome! :+1: 

We would be definitely interested in format validator, feel free to contribute when it's done :)

**Comment by michalmarcinkowski** [comment]
@malukenho could you run the formatter once again? We will merge this and right after that #4079 to make sure everything that goes into master is formatted properly.

**Comment by malukenho** [comment]
@michalmarcinkowski rebased and CS issues fixed. When It's merged I have to rebase the other PR.

**Comment by pjedrzejewski** [comment]
Thank you @malukenho! Nice work, go ahead and rebase other PR, I will merge it then. :+1:

**Comment by pamil** [comment]
@malukenho is there any way to allow for multi-line tags definitions? Like here: https://travis-ci.org/pamil/Sylius/jobs/109572562#L1400-L1402

We're introducing `@todo` tag and the diffs will be cleaner if this tag is defined in another line.

**Comment by malukenho** [comment]
@pamil I'm working in a way to make format rules configurable. In one or two weeks I should have it done :/

**Pull request merged**